Dockerfile

```diff
@@ -1,12 +1,13 @@
-# The base go-image
-FROM golang:1.21-alpine
+# Builder image
+FROM golang:1.21-alpine as build
 
 RUN mkdir /app 
 COPY . /app
-
 WORKDIR /app
+RUN go build
```

Comment:
Do you mind adding back the `-v`? This lets us see the build progress in real time.